sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPEncryptionTests.cs

```diff
@@ -30,16 +30,18 @@ public void DiskEncryptionPositiveTest()
             using (MockContext context = MockContext.Start(this.GetType()))
             {
                 EnsureClientsInitialized(context);
-                string testVaultId = @"/subscriptions/0296790d-427c-48ca-b204-8b729bbd8670/resourceGroups/longrunningrg-centraluseuap/providers/Microsoft.KeyVault/vaults/swaggerKeyVault5";
-                string encryptionKeyUri = @"https://swaggerkeyvault5.vault.azure.net/keys/swaggerKey/52317b056a2c49d0b8673a67d072655e";
-                string secretUri = @"https://swaggerkeyvault5.vault.azure.net/secrets/swaggerSecret/0e01a1ab339e40ff9f01ea98ae8ee6b5";
+                string testVaultId = @"/subscriptions/e37510d7-33b6-4676-886f-ee75bcc01871/resourceGroups/RGforSDKtestResources/providers/Microsoft.KeyVault/vaults/testingkvsdk";
```

Comment:
This pattern is highly discouraged.  It is best if tests do their own setup, bothto minimize code changes when updating the api-version and to reduce the amount of knowledge you need to have about the tests before running them live.

Reply:
@markcowl , agreed, we were not in charge of these tests and don't currently have the knowledge to change it. The DiskRP team needs to modify this. We will be curating these tests in the future.